renderer/components/audio-player/audio-player.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState, useCallback, useEffect, useRef } from 'react';
+import styled from '@emotion/styled';
+import WaveSurfer from 'wavesurfer.js';
+import { Text } from 'rebass';
+import useTheme from '../../hooks/useTheme';
+import type { Theme } from '../../theme';
+import useAppState from '../../hooks/useAppState';
+import getFileName from '../../../util/getFileName';
+import PauseButton from './pause-button';
+import withWave from './wave-surfer';
+
+import { Sound } from '../../../@types';
+
+const WaveContainer = styled.div`
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: center;
+    height: 70px;
+    width: 100%;
+`;
+
+const WaveForm = withWave(
+    ({ waveColor, togglePause, playing }: { waveColor?: string; togglePause: () => void; playing: boolean }) => {
+        return <PauseButton color={waveColor} onClick={togglePause} playing={playing} />;
+    }
+);
+
+const AudioPlayer = ({
+    sound,
+    waveColor: initialWaveColor,
+    progressColor: initialProgressColor,
+}: {
+    sound: Sound;
+    waveColor?: string;
+    progressColor?: string;
+}) => {
+    const { dispatch, state } = useAppState();
```

Comment:
No need to destruct `state` here if it isn't used in this file.

Reply:
```suggestion
    const { dispatch } = useAppState();
```
Good catch, making a suggestion to fix this